opentelekomcloud/data_source_opentelekomcloud_vpc_bandwidth.go

```diff
@@ -0,0 +1,113 @@
+package opentelekomcloud
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/networking/v1/bandwidths"
+)
+
+func dataSourceBandWidth() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceBandWidthRead,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"size": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validateIntegerInRange(5, 2000),
```

Comment:
we don't need to create our own valdation, because we have `validation.IntBetween(5, 2000)`